js/screens/welcome-screen.js

```diff
@@ -3,3 +3,51 @@ export const welcomeScreen = () => {
    welcomeStepWrapper.innerText = 'Welcome screen';
    return welcomeStepWrapper;
 };
+
+function orangeButton() {
+   const chooseButton = document.createElement('button');
+   const contentButton = document.innerText('Choose');
+   const srcButton = document.src('./components/choose.svg');
+   document.chooseButton.appendChild(contentButton && srcButton);
+   return chooseButton;
+}
+
+/*
+button.addEventListener('click', function (event) {});
+*/
+
+image.onClick = function Choose() {
+   image.src = './components/choose.svg';
+   return image.src;
+}
+
+function UltimatePixelChampionships(fontWelcomeScreen, fontForm) {
+   const fontWelcomeScreen = document.createElement('div');
+   fontWelcomeScreen.innerText = src('./assets/Ultimate Pixel Championships.svg');
+   const fontForm = document.createElement('div');
+   fontForm.innerText = src('./assets/fill-Out-Form.svg')
+   return fontWelcomeScreen && fontForm;
+}
+
+function lineBelowUPC(orangeLine) {
+   const orangeLine = document.createElement('div');
+   orangeLine = src('./components/Line 15.svg');
+   return orangeLine;
+}
+
+function margins(marginNav,marginFooter) {
+   const marginNav = document.createElement('div');
+   marginNav = src('./components/Margin-nav.svg');
```

Comment:
**nitpick:** If you are setting `src` you should use `img` tag instead of `div` to keep up with semantics